cypress/e2e/e2e.cy.js

```diff
@@ -0,0 +1,117 @@
+describe('Teste E2E EngageSphere', () => {
+    beforeEach(() => {
+        cy.visit('/')
+        cy.contains('button', 'Accept').click();
+    })
+
+    it('Retorna à lista de clientes ao clicar no botão "Voltar"', () => {
+        cy.get('button[aria-label="View company: Lowe Co"]').click();
+        cy.contains('button', 'Back').should('be.visible')
```

Comment:
Em vez de encontrar o botão _Back_ de novo com o `cy.contains()`, você poderia ter encadeado um `.clic()` direto ao `.should('be.visible'), usando um comando a menos.

Assim:

`cy.contains('button', 'Back').should('be.visible').click()`